src/GraphQLParser/AST/Values/GraphQLValue.cs

```diff
@@ -7,4 +7,8 @@ namespace GraphQLParser.AST;
 /// </summary>
 public abstract class GraphQLValue : ASTNode
 {
+    /// <summary>
+    /// Returns the CLR object that represents the value of this AST node.
+    /// </summary>
+    public abstract object? ClrValue { get; }
```

Comment:
All value nodes now can return CLR representation of the parsed value. Some nodes have backing fields, some not (NullValue, BooleanValue, EnumValue).

Reply:
+NameNode of course (not a value node)

Reply:
Reverted.